scripts/helpers/db-command-helpers.ts

```diff
@@ -126,6 +127,8 @@ export const runResetQueries = async (
       // When owner is superuser - this is not needed, but in deployed environments and for test databases owner is a regular user and also needs such grant.
       `GRANT ${dbGqlRole} TO ${dbOwner};`,
       createEnvOwner ? `GRANT ${dbGqlRole} TO ${dbEnvOwner};` : '',
+      // By granting owner_role to the root role, we can then create the database while settin owner_role as the owner.
```

Comment:
I would change this to something that explains why the root user even needs such a grant in the first place. e.g. `In some systems (e.g. --FSPG--), it is not possible to get the actual ROOT user credentials, so a user with a lot of privileges is used instead and we need to grant db owner rights to it.`

`--FSPG--` should be replaced with something understandable for regular users, even I have not found what it stands for. Maybe replace with `hosted Azure or AWE database servers` or something technically-correct along those lines.